jenkins/scripts/VersionSelectorScript.groovy

```diff
@@ -51,6 +51,7 @@ def buildExclusions = [
 
   // PPC BE ------------------------------------------------
   [ /^ppcbe-ubuntu/,                  anyType,     gte(8)  ],
+  [ /^ppcle-ubuntu/,                  anyType,     gte(12) ],
```

Comment:
This should only go in once we are ready to transition from unbuntu to centos for ppce le.  To start it should instead exclude running centos on versions lower than 12.x.  The result should be builds on both ubuntu and centos for 12.x and above.

Reply:
`[ /^centos7/,                       releaseType, lt(12)  ],` is already in the file, so I should just change that to `anyType`, and remove the line I added, and we'll be good?

Reply:
that was jsut changed to centos7-64, it was catching arm64 which it shouldn't, so you should be specific

Reply:
I backed this out.